packages/fiori/src/ViewSettingsDialog.ts

```diff
@@ -624,6 +632,7 @@ class ViewSettingsDialog extends UI5Element {
 			sortBy,
 			sortByItem,
 			filters: this.selectedFilters,
+			filterItems: this.selectedFilterItems,
```

Comment:
Is this new field really necessary? Can't the app do this if they need such information? Or is this code required by the control somewhere?

Reply:
The app cannot do this themself because until now we only return the text of the provided filters in the details, therefore we would have to provide the `ui5-filter-item`, and its `ui5-filter-item-option`'s in the event details.
This way providing for example `data-key="filterText1"` to the `ui5-filter-item` or `ui5-filter-item-option` would make them able to be accessed by `$0.dataset.key` HTML Property.